run.sh

```diff
@@ -6,4 +6,18 @@ fi
 # make rabbit own its own files
 chown -R rabbitmq:rabbitmq /var/lib/rabbitmq
 
-exec /usr/sbin/rabbitmq-server
+if [ -z "$CLUSTER_WITH" ] ; then
+    /usr/sbin/rabbitmq-server
+else
+    if [ -f /.CLUSTERED ] ; then
+        /usr/sbin/rabbitmq-server
+    else
+        touch /.CLUSTERED
+        /usr/sbin/rabbitmq-server -detached
+        rabbitmqctl stop_app
+        rabbitmqctl join_cluster rabbit@$CLUSTER_WITH
+        rabbitmqctl start_app
+        tail -f /var/log/rabbitmq/rabbit\@$HOSTNAME.log
```

Comment:
Is there any way to run rabbitmq-server as a foreground process, so that the container can stop if the the process is terminated. If not, I would rather use supervisor to start the server, at least it can try to restart the server on failure.

BTW, `tail -f` doesn't deal with the log rotation, etc. `-F` option will do the retry on error.